pom.xml

```diff
@@ -2355,8 +2355,6 @@ under the License.
 								<exclude>@org.apache.flink.annotation.Experimental</exclude>
 								<exclude>@org.apache.flink.annotation.Internal</exclude>
 								<!-- MARKER: start exclusions; these will be wiped by tools/releasing/update_japicmp_configuration.sh -->
-								<!-- New method has been added to the interface, but this interface shouldn't be implemented by users, only used/called, so this doesn't break API compatibility. -->
-								<exclude>org.apache.flink.api.common.eventtime.WatermarkGeneratorSupplier$Context</exclude>
 								<!-- Base interface has been extracted from the existing Clock class. This interface shouldn't affect user code. -->
 								<exclude>org.apache.flink.util.clock.Clock</exclude>
```

Comment:
I'm keeping this incompatibility excluded as is. I think/hope it should be fine.